slash/utils/interactive.py

```diff
@@ -50,15 +50,19 @@ def generate_interactive_test():
     return returned
 
 @contextmanager
-def notify_if_slow_context(message, slow_seconds=1):
+def notify_if_slow_context(message, slow_seconds=1, end_message=None):
     evt = threading.Event()
+    evt.should_report_end_msg = False
     def notifier():
         if not evt.wait(timeout=slow_seconds) and context.session is not None:
             context.session.reporter.report_message(message)
+            evt.should_report_end_msg = end_message is not None
```

Comment:
@ayalash this line should just set `should_report_end_msg` to `True`. The check for `None` should be made in the end IMO